schema/schema.yaml

```diff
@@ -254,6 +264,41 @@ tables:
         - "#Publications.reference"
 
 
+  - name: ParameterList
+    "@id": "#ParameterList"
+    description: Parameters lookup table
+    primaryKey: 
+    - "#ParameterList.parameter"
+
+    columns:
+    - name: parameter
+      "@id": "#ParameterList.parameter"
+      datatype: string
+      length: 30
+      description: Main identifier for a parameter
+      ivoa:ucd: meta.id;meta.main
+      nullable: false
+    - name: description
+      "@id": "#ParameterList.description"
+      datatype: string
+      length: 1000
```

Comment:
Should this be 100 characters to be consistent with other description fields?
We should also consider using the ivoa:ucd type `meta.note` for this and comments fields (see #120)